docs/docs/Components/components-processing.md

```diff
@@ -114,6 +114,7 @@ Replace `YOUR_FLOW_ID` with your flow ID.
 ```bash
 curl -X POST "http://localhost:7860/api/v1/webhook/YOUR_FLOW_ID" \
 -H 'Content-Type: application/json' \
+-H 'x-api-key: LANGFLOW_API_KEY' \
```

Comment:
Are these placeholders or variables on this page?

Reply:
Placeholders - variables are only in the api section

Reply:
Added Replace `LANGFLOW_SERVER_ADDRESS`, `FLOW_ID`, and `LANGFLOW_API_KEY` with the values from your Langflow deployment.